packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -147,3 +147,35 @@ function valueToAttributeValue(
       return {};
   }
 }
+
+const HTTP_ATTRIBUTE_MAPPING: {[key: string]: string} = {
+  'http.method': '/http/method',
+  'http.url': '/http/url',
+  'http.target': '/http/target',
+  'http.host': '/http/host',
+  'http.scheme': '/http/client_protocol',
+  'http.status_code': '/http/status_code',
+  'http.flavor': '/http/flavor',
+  'http.user_agent': '/http/user_agent',
+  'http.request_content_length': '/http/request_content_length',
+  'http.request_content_length_uncompressed':
+    '/http/request_content_length_uncompressed',
+  'http.response_content_length': '/http/response_content_length',
+  'http.response_content_length_uncompressed':
+    '/http/response_content_length_uncompressed',
```

Comment:
not in https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects.traces#get

In the python impl we mapped these to `/http/(request|response)/size` https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/blob/c3772de4cf50950f1d51685ff3bbf0e46ee026df/opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py#L373-L374